### PR TITLE
Add node label_present/absent states functions

### PR DESCRIPTION
**Pull request by toopy**

### CODE REVIEW

**Comment by toopy** [comment]
@bersace opm ?

File: _states/jenkins_node.py [hunk 1]
```diff
@@ -115,3 +118,116 @@ def absent(name):
...
+
+def label_present(name, node):
+    """Ensure jenkins label is present in a given node.
+
+    name
+        The name of the label to be present.
+
+    view
+        The target node.
+    """
```
Comment (bersace) [inline _states/jenkins_node.py]: Please use `label_present(name, labels`)

File: _states/jenkins_node.py [hunk 2]
```diff
@@ -115,3 +118,116 @@ def absent(name):
         'new': 'absent',
     }
     return ret
+
+
+def label_present(name, node):
```
Comment (bersace) [inline _states/jenkins_node.py]: use `label_present(name, label)`

File: _states/jenkins_node.py [hunk 3]
```diff
@@ -115,3 +118,116 @@ def absent(name):
...
+    ret['changes'] = {
+        'old': old,
+        'new': new,
+    }
+
+    ret['result'] = None if test else True
+    return ret
+
+
+def label_absent(name, node):
```
Comment (bersace) [inline _states/jenkins_node.py]: Use `label_absent(name, label)`.

**Comment by toopy** [comment]
@bersace opm ?

**Comment by bersace** [comment]
:+1:

**Pull request merged**